modules/lambdas/variables.tf

```diff
@@ -151,16 +151,28 @@ variable "orca_recovery_retrieval_type" {
 }
 
 variable "orca_sqs_staged_recovery_queue_arn" {
-  type        = "string"
+  type        = string
   description = "The ARN of the staged-recovery-queue SQS"
 }
 
 variable "orca_sqs_status_update_queue_id" {
   type        = string
-  description = "The URL of the SQS queue that recoery status updates are read from/posted to."
+  description = "The URL of the SQS queue that recovery status updates are read from/posted to."
+}
+
+variable "orca_sqs_recovery_queue_id" {
```

Comment:
Make sure this new variable gets bubbled up to the orca/main.tf, orca/variables.tf and the main.tf and variables.tf files if needed.